test/parallel/test-vm-getter-throw.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+
+const vm = require('vm');
+
+const x = {};
+
+const code = `
+  var foo = 1;
+  Object.defineProperty(this, "bar", {
+    enumerable: true,
+    get: function () { throw 42 }
+  });
+  var baz = 2;
+`;
+vm.runInNewContext(code, x);
+
+assert.deepEqual(x, { foo: 1, baz: 2 });
```

Comment:
I think this should be 
`assert.deepEqual(x, { foo: 1, bar: get() {...}, baz: 2 });`

If you don't want to change the implementation, can we make this a known issue? I.e., split this up in a test and a known issue or at least leave a comment that `x` should have the property `bar`?